### PR TITLE
XS✔ ◾ Adding prefix/ suffix to all Accounting Rules & reorder Accounting catagory

### PR DESCRIPTION
**Pull request by JimmyChenSSW**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

As per Adam's email subject: 👑 Rules to Better Accounting | SSW.Rules
Add prefixes and suffixes to make them easier to find.

> 2. What was changed?

- Add Suffix - Be consistent and add “(Aussie Only)”
- Add Prefix - e.g., “Payroll -“
- Reorder the category to tie the same prefix together and list with priority

> 3. Did you do pair or mob programming (list names)?

No - just myself.

### CODE REVIEW

File: rules/do-you-calculate-payroll-tax-correctly/rule.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 seoDescription: Calculate payroll tax correctly by understanding Australian government rules and state-specific exemptions, avoiding common errors and staying up to date with changes.
 type: rule
 archivedreason:
-title: Do you calculate Payroll tax (Australia) correctly?
+title: Paryoll - Do you calculate Payroll tax correctly? (Aussie Only)
```
Comment (tiagov8) [inline rules/do-you-calculate-payroll-tax-correctly/rule.md:5]: Hi @JimmyChenSSW 

On all "Payroll" you made a typo

- [ ] 1. Please fix

File: rules/tax-invoice-vs-eftpos-receipt/rule.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 seoDescription: Understand the difference between Tax Invoice and Eftpos Receipt - A comprehensive guide to tax compliance and expense management.
 type: rule
-title: "Do you understand the difference: Tax Invoice vs. Eftpos Receipt?"
+title: Expenses - Do you understand the difference: Tax Invoice vs. Eftpos Receipt?
```
Comment (tiagov8) [inline rules/tax-invoice-vs-eftpos-receipt/rule.md:4]: Hi @JimmyChenSSW 

Not a big fan of a colon in the title

- [ ] 1. Please change to "Do you understand the difference between Tax Invoice and Eftpos Receipt?"

File: rules/salary-sacrifice-electronic-devices/rule.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 seoDescription: Pay less tax on electronic devices by salary sacrificing with your pre-tax money and save approximately 30% depending on your tax bracket.
 type: rule
-title: Do you take advantage of salary sacrificing electronic devices?
+title: Paryoll - Do you take advantage of salary sacrificing electronic devices? (Aussie Only)
```
Comment (tiagov8) [inline rules/salary-sacrifice-electronic-devices/rule.md:4]: Hi @JimmyChenSSW 

On all "(Aussie only)" the word "only" should be in lowercase

- [ ] 1. Please fix all

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: rules/do-you-calculate-payroll-tax-correctly/rule.md (hunk 1 above)
Comment (JimmyChenSSW) [reply rules/do-you-calculate-payroll-tax-correctly/rule.md]: Done - That was a stupid error play around with my Copy+Paste.
Thank you Tiago!

File: rules/tax-invoice-vs-eftpos-receipt/rule.md (hunk 2 above)
Comment (JimmyChenSSW) [reply rules/tax-invoice-vs-eftpos-receipt/rule.md]: Done - Agree.

File: rules/salary-sacrifice-electronic-devices/rule.md (hunk 3 above)
Comment (JimmyChenSSW) [reply rules/salary-sacrifice-electronic-devices/rule.md]: Done - thank you!

**Comment by JimmyChenSSW** [comment]
Hi @tiagov8 , thanks for your help.
All Done.

**Review by tiagov8 — APPROVED** [review]

**Comment by tiagov8** [comment]
Thanks @JimmyChenSSW 

Approved ✅

**Pull request merged**